pkg/webhook/node/plugins/cpunormalization/node_cpu_normalization.go

```diff
@@ -0,0 +1,120 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cpunormalization
```

Comment:
Actually, this feature should go to node resource amplification rather than CPU normalization, thus the webhook should refer to the amplification ratio rather than normalization ratio to do amplification.

However, there's one thing that we haven't done there: The CPU normalization controller should multiply the normalization ratio to the original amplification ratio (or just annotate it if not exists). I can fire another separate PR to complete this.